src/builder/guard.rs

```diff
@@ -46,6 +46,7 @@ impl<'a> Guard<'a> {
 
     pub fn start(&mut self) -> Result<(), String> {
         mount_system_dirs()?;
+        mount_run(&Path::new("/vagga/root/run"))?;
```

Comment:
In order to not mount `/run` twice when somebody sets volume into `/run`. See https://github.com/tailhook/vagga/pull/377/commits/aa13159c8266885555e28f17115dc56b62cf9bb1#diff-2713dfcee40c652f64f1994c5200e045L408